android/envoy/build.gradle

```diff
@@ -54,13 +54,16 @@ dependencies {
     implementation 'androidx.core:core-ktx:1.12.0'
     implementation 'androidx.preference:preference-ktx:1.2.1'
     implementation 'androidx.localbroadcastmanager:localbroadcastmanager:1.1.0'
+    implementation 'org.greatfire.envoy:cronet:108.0.5359.195'
     //implementation 'org.greatfire.envoy:cronet:107.0.5304.150-1'
-    implementation 'org.greatfire:IEnvoyProxy:1.5.0'
+    // implementation 'org.greatfire:IEnvoyProxy:1.5.0'
 
-    debugApi(name: 'cronet-debug', ext: 'aar')
-    releaseApi(name: 'cronet-release', ext: 'aar')
+    // debugApi(name: 'cronet-debug', ext: 'aar')
+    // releaseApi(name: 'cronet-release', ext: 'aar')
     // debugApi(name: 'IEnvoyProxy', ext: 'aar')
     // releaseApi(name: 'IEnvoyProxy', ext: 'aar')
+    debugApi(name: 'plenipotentiary', ext: 'aar')
+    releaseApi(name: 'plenipotentiary', ext: 'aar')
 }
```

Comment:
i assume we can delete the old cronet gradle dependency and all ienvoyproxy depenencies.  maybe that can wait until there's a gradle plenipotentiary dependency to add.